res/layout/game_activity.xml

```diff
@@ -1,21 +1,45 @@
 <?xml version="1.0" encoding="utf-8"?>
-<merge xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This is the main thing I'm worried about. Comments in the source seem to imply that gamepad support is very finicky in Android and highly dependent on the structure of the view hierarchy, so we need to make sure it still works!